src/init.py

```diff
@@ -0,0 +1,24 @@
+import os
+import logging
+import json
+
+def initialize_logger():
+    log_level = getattr(logging, os.getenv('LOG_LEVEL', default='WARN').upper(), None)
+    logging.basicConfig(level=log_level, format='%(levelname)s:%(name)s:%(asctime)s %(message)s')
+
+def load_configuration():
```

Comment:
Is this loading configuration as json? if it is you should look at json.load(). It loads json from a file into an object